keras_nlp/models/task_test.py

```diff
@@ -69,6 +69,9 @@ def test_from_preset_errors(self):
         with self.assertRaises(ValueError):
             # No loading on an incorrect class.
             BertClassifier.from_preset("gpt2_base_en", load_weights=False)
+        with self.assertRaises(FileNotFoundError):
```

Comment:
In general as a best practice unit tests should use `assertRaisesRegex`, because sometimes a *different* exception with the same type can be raised (not the one you expect). In this case it doesn't matter because the exception class is fairly specific.

Reply:
Make sense! I updated these unit tests to use `assertRaisesRegex`.